jsonschema-core.xml

```diff
@@ -2324,61 +2324,49 @@
                     properties and array items, and combining their results.
                 </t>
                 <section title="Keywords for Applying Subschemas to Arrays">
-                    <section title="items">
+                    <section title="prefixItems">
                         <t>
-                            The value of "items" MUST be either a valid JSON Schema or
-                            an array of valid JSON Schemas.
+                            The value of "prefixItems" MUST be an array of valid JSON Schemas.
                         </t>
                         <t>
-                            If "items" is a schema, validation succeeds if all elements
-                            in the array successfully validate against that schema.
-                        </t>
-                        <t>
-                            If "items" is an array of schemas, validation succeeds if
-                            each element of the instance validates against the schema at the
-                            same position, if any.
+                            Validation succeeds if each element of the instance validates
+                            against the schema at the same position, if any.
                         </t>
                         <t>
                             This keyword produces an annotation value which is the largest
                             index to which this keyword applied a subschema.  The value
                             MAY be a boolean true if a subschema was applied to every
-                            index of the instance, such as when "items" is a schema.
-                        </t>
-                        <t>
-                            Annotation results for "items" keywords from multiple
-                            schemas applied to the same instance location are combined
-                            by setting the combined result to true if any of the values
-                            are true, and otherwise retaining the largest numerical value.
+                            index of the instance, such as is produced by the "items" keyword.
```

Comment:
it seems simpler to just have `prefixItems` produce a max index annotation, and `items` produce true.

Reply:
also the language in `items` doesn't seem to account for a `true` annotation result from `prefixItems`, only numeric

Reply:
@notEthan if `prefixItems` is true then `items` doesn't do anything.  There's no need for a length check.